### ISSUE TITLE
Bed Bug

### ISSUE THREAD

**Issue by 112938576mjs** [THEREALWWEFAN231/wwe#3]
**WWE version:** 2.5
**Minecraft version:** 1.13.2
**Operating system:** Win10
**Optifine/Other mods:** Optifine

**Whats the issue:** Right clicking a bed at any time forces you to go to a "Saving world" screen and then crashes your game. (This is happening in realms for me)

**Crash-report(Would be very useful also please format):** 

```
`---- Minecraft Crash Report ----
// Ouch. That hurt :(

Time: 09/04/19 12:29
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at baritone.f.onBlockInteract(Unknown Source)
	at baritone.y.a(Unknown Source)
	at baritone.y$$Lambda$1220/991179543.accept(Unknown Source)
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:890)
	at baritone.y.onBlockInteract(Unknown Source)
	at cft.handler$onBlockUse$zde000(SourceFile)
	at cft.au(SourceFile:1179)
	at cft.aw(SourceFile:1410)
	at cft.m(SourceFile:1251)
	at cft.c(SourceFile:821)
	at cft.a(SourceFile:395)
	at net.minecraft.client.main.Main.main(SourceFile:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at baritone.f.onBlockInteract(Unknown Source)
	at baritone.y.a(Unknown Source)
	at baritone.y$$Lambda$1220/991179543.accept(Unknown Source)
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:890)
	at baritone.y.onBlockInteract(Unknown Source)
	at cft.handler$onBlockUse$zde000(SourceFile)
	at cft.au(SourceFile:1179)
	at cft.aw(SourceFile:1410)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [ctj['y0slaap'/0, l='MpServer', x=-454.58, y=48.00, z=39.29]]
	Chunk stats: MultiplayerChunkCache: 361, 361
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: {}
	Level spawn location: World: (-16,66,144), Chunk: (at 0,4,0 in -1,9; contains blocks -16,0,144 to -1,255,159), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 236043 game time, 26386 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 76 total; [ctj['y0slaap'/0, l='MpServer', x=-454.58, y=48.00, z=39.29], ajl['Bat'/6912, l='MpServer', x=-515.13, y=19.10, z=0.75], ajl['Bat'/6913, l='MpServer', x=-525.53, y=19.10, z=0.12], ann['Skeleton'/6914, l='MpServer', x=-517.50, y=19.00, z=38.50], akd['Rabbit'/6921, l='MpServer', x=-506.54, y=64.00, z=-7.97], ajl['Bat'/6922, l='MpServer', x=-507.25, y=17.07, z=14.54], ann['Skeleton'/6923, l='MpServer', x=-496.50, y=17.00, z=59.50], amt['Creeper'/6924, l='MpServer', x=-496.30, y=21.00, z=61.70], anw['Zombie'/6925, l='MpServer', x=-499.45, y=23.00, z=62.76], ann['Skeleton'/6926, l='MpServer', x=-496.35, y=17.00, z=67.23], ann['Skeleton'/6930, l='MpServer', x=-486.50, y=18.00, z=-17.50], anq['Spider'/6931, l='MpServer', x=-483.96, y=18.00, z=-13.57], amw['Enderman'/6932, l='MpServer', x=-481.49, y=18.00, z=-15.40], amt['Creeper'/6933, l='MpServer', x=-492.50, y=51.00, z=14.50], ajt['Cow'/6934, l='MpServer', x=-490.15, y=64.00, z=12.36], aka['Pig'/6935, l='MpServer', x=-484.52, y=65.00, z=36.73], anw['Zombie'/6936, l='MpServer', x=-490.30, y=23.00, z=52.70], anw['Zombie'/6937, l='MpServer', x=-494.56, y=21.00, z=62.99], amt['Creeper'/6938, l='MpServer', x=-493.30, y=21.00, z=63.70], anw['Zombie'/6939, l='MpServer', x=-493.30, y=21.00, z=67.30], anw['Zombie'/6940, l='MpServer', x=-493.96, y=21.00, z=67.53], akf['Sheep'/6941, l='MpServer', x=-484.78, y=66.00, z=78.81], ann['Skeleton'/6942, l='MpServer', x=-490.44, y=23.00, z=108.32], amt['Creeper'/6947, l='MpServer', x=-475.50, y=31.00, z=-2.50], amt['Creeper'/6948, l='MpServer', x=-478.16, y=32.00, z=-5.47], akq['Horse'/6949, l='MpServer', x=-460.56, y=63.00, z=50.78], akq['Horse'/6950, l='MpServer', x=-482.71, y=64.00, z=46.95], ann['Skeleton'/6951, l='MpServer', x=-464.50, y=34.00, z=107.50], amt['Creeper'/6953, l='MpServer', x=-465.21, y=19.00, z=117.46], ann['Skeleton'/6960, l='MpServer', x=-454.71, y=44.00, z=-37.49], amt['Creeper'/6961, l='MpServer', x=-455.30, y=44.00, z=-36.32], amt['Creeper'/6962, l='MpServer', x=-460.81, y=29.00, z=-30.47], anw['Zombie'/6963, l='MpServer', x=-451.53, y=23.00, z=-6.70], amt['Creeper'/6964, l='MpServer', x=-458.50, y=20.00, z=76.50], ann['Skeleton'/6965, l='MpServer', x=-461.50, y=34.00, z=106.50], amt['Creeper'/6971, l='MpServer', x=-435.50, y=28.00, z=-36.50], apl['Minecart with Chest'/6972, l='MpServer', x=-440.49, y=24.00, z=-20.49], akm['Wolf'/6973, l='MpServer', x=-437.30, y=41.00, z=35.30], amm['Dirt'/6974, l='MpServer', x=-435.54, y=40.00, z=36.88], ann['Skeleton'/6975, l='MpServer', x=-443.51, y=18.00, z=67.79], ann['Skeleton'/6976, l='MpServer', x=-439.41, y=16.00, z=78.70], apl['Minecart with Chest'/6978, l='MpServer', x=-423.50, y=28.06, z=-22.50], ajl['Bat'/6979, l='MpServer', x=-429.25, y=26.10, z=-29.60], ajl['Bat'/6980, l='MpServer', x=-424.36, y=33.82, z=-14.12], ajl['Bat'/6981, l='MpServer', x=-420.83, y=38.11, z=55.11], ajl['Bat'/6986, l='MpServer', x=-405.75, y=26.10, z=-29.25], amt['Creeper'/6987, l='MpServer', x=-412.58, y=17.00, z=84.15], amt['Creeper'/6988, l='MpServer', x=-413.57, y=17.00, z=83.92], amt['Creeper'/6989, l='MpServer', x=-415.16, y=17.00, z=88.41], akd['Rabbit'/6990, l='MpServer', x=-401.44, y=65.00, z=89.37], ann['Skeleton'/6991, l='MpServer', x=-400.48, y=28.00, z=104.72], akd['Rabbit'/6992, l='MpServer', x=-406.08, y=67.00, z=103.37], akd['Rabbit'/6993, l='MpServer', x=-403.48, y=69.00, z=105.46], anw['Zombie'/6994, l='MpServer', x=-384.82, y=27.00, z=45.52], anw['Zombie'/6995, l='MpServer', x=-393.50, y=23.00, z=37.50], anw['Zombie'/6996, l='MpServer', x=-399.49, y=20.00, z=30.71], ann['Skeleton'/6997, l='MpServer', x=-395.50, y=23.00, z=63.50], ann['Skeleton'/6998, l='MpServer', x=-389.38, y=23.00, z=67.30], ann['Skeleton'/6999, l='MpServer', x=-395.50, y=24.00, z=59.71], amw['Enderman'/7000, l='MpServer', x=-394.50, y=19.00, z=59.50], ajl['Bat'/7001, l='MpServer', x=-383.50, y=23.97, z=37.53], ann['Skeleton'/7002, l='MpServer', x=-392.54, y=36.00, z=56.27], amw['Enderman'/7003, l='MpServer', x=-390.44, y=23.00, z=68.54], amt['Creeper'/7004, l='MpServer', x=-395.17, y=16.00, z=95.47], akd['Rabbit'/7005, l='MpServer', x=-393.23, y=67.00, z=89.50], akd['Rabbit'/7006, l='MpServer', x=-397.31, y=64.00, z=91.51], ajl['Bat'/7007, l='MpServer', x=-394.92, y=17.10, z=97.19], amt['Creeper'/7009, l='MpServer', x=-383.50, y=23.00, z=69.50], anw['Zombie'/6901, l='MpServer', x=-530.39, y=22.00, z=27.30], ann['Skeleton'/6905, l='MpServer', x=-526.50, y=29.00, z=-32.50], ajl['Bat'/6906, l='MpServer', x=-523.71, y=18.03, z=-1.48], ajl['Bat'/6907, l='MpServer', x=-525.51, y=16.10, z=-0.46], ajl['Bat'/6908, l='MpServer', x=-516.22, y=37.34, z=-2.48], ajl['Bat'/6909, l='MpServer', x=-532.58, y=31.32, z=-24.87], ajl['Bat'/6910, l='MpServer', x=-515.25, y=20.10, z=3.52], ajl['Bat'/6911, l='MpServer', x=-514.13, y=23.10, z=1.75]]
	Retry entities: 0 total; []
	Server brand: vanilla
	Server type: Non-integrated multiplayer server
Stacktrace:
	at crg.a(WorldClient.java:576)
	at cft.b(SourceFile:1815)
	at cft.a(SourceFile:408)
	at net.minecraft.client.main.Main.main(SourceFile:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.13.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 570317600 bytes (543 MB) / 1543503872 bytes (1472 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: wwe 1.13.2
	LWJGL: 3.1.6 build 14
	OpenGL: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 411.63, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 6x AMD FX(tm)-6300 Six-Core Processor 
	OptiFine Version: OptiFine_1.13.2_HD_U_E6
	OptiFine Build: 20190214-140229
	Render Distance Chunks: 48
	Mipmaps: 4
	Anisotropic Filtering: 16
	Antialiasing: 0
	Multitexture: true
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 411.63
	OpenGlRenderer: GeForce GTX 1060 6GB/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 6`

```

**Comment by THEREALWWEFAN231** [comment]
Ok, ill look into it

**Comment by THEREALWWEFAN231** [comment]
Did you have baritone going or did you run baritone in that game instance?

**Comment by THEREALWWEFAN231** [comment]
What modules did you have active, maybe send a screen shot or something, i wasent able to recreate this

**Comment by 112938576mjs** [comment]
I tried it with all modules turned off in the GUI. I'm not sure about Baritone though and I don't know how to tell if it's running or not

**Comment by THEREALWWEFAN231** [comment]
Could you send the crash log when exiting the nether, with that we might be able to find something in common thats crashing, it has something to due with baritone, even if you dont have it on

**Comment by 112938576mjs** [comment]
Here's the crash log for exiting the nether:

`---- Minecraft Crash Report ----
// I feel sad now :(

Time: 11/04/19 11:18
Description: Ticking entity

java.lang.NullPointerException: Ticking entity
	at axy.w(SourceFile:2348)
	at aer.p(SourceFile:1010)
	at aer.ap(SourceFile:1023)
	at afa.W(SourceFile:313)
	at afb.W(EntityLiving.java:274)
	at aer.R_(SourceFile:377)
	at afa.R_(SourceFile:1757)
	at afb.R_(EntityLiving.java:382)
	at ajl.R_(SourceFile:113)
	at axy.a(SourceFile:1243)
	at axy.g(SourceFile:1218)
	at axy.o_(SourceFile:1100)
	at cft.m(SourceFile:1281)
	at cft.c(SourceFile:821)
	at cft.a(SourceFile:395)
	at net.minecraft.client.main.Main.main(SourceFile:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at axy.w(SourceFile:2348)
	at aer.p(SourceFile:1010)
	at aer.ap(SourceFile:1023)
	at afa.W(SourceFile:313)
	at afb.W(EntityLiving.java:274)
	at aer.R_(SourceFile:377)
	at afa.R_(SourceFile:1757)
	at afb.R_(EntityLiving.java:382)
	at ajl.R_(SourceFile:113)
	at axy.a(SourceFile:1243)
	at axy.g(SourceFile:1218)

-- Entity being ticked --
Details:
	Entity Type: minecraft:bat (ajl)
	Entity ID: 1606
	Entity Name: Bat
	Entity's Exact location: -485.11, 30.10, -11.40
	Entity's Block location: World: (-486,30,-12), Chunk: (at 10,1,4 in -31,-1; contains blocks -496,0,-16 to -481,255,-1), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at axy.o_(SourceFile:1100)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [ctj['y0slaap'/0, l='MpServer', x=-412.95, y=70.98, z=37.76]]
	Chunk stats: MultiplayerChunkCache: 86, 86
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: {}
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 509958 game time, 300301 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: true), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 7 total; [ctj['y0slaap'/0, l='MpServer', x=-412.95, y=70.98, z=37.76], ajl['Bat'/1606, l='MpServer', x=-485.11, y=30.10, z=-11.40], amt['Creeper'/1608, l='MpServer', x=-492.50, y=51.00, z=28.50], anw['Zombie'/1609, l='MpServer', x=-492.50, y=51.00, z=32.50], ani['Zombie Pigman'/1506, l='MpServer', x=-88.34, y=96.00, z=-8.53], ani['Zombie Pigman'/1518, l='MpServer', x=-112.50, y=74.00, z=30.50], ani['Zombie Pigman'/1526, l='MpServer', x=-118.53, y=102.00, z=-47.53]]
	Retry entities: 3 total; [ani['Zombie Pigman'/1506, l='MpServer', x=-88.34, y=96.00, z=-8.53], ani['Zombie Pigman'/1526, l='MpServer', x=-118.53, y=102.00, z=-47.53], ani['Zombie Pigman'/1518, l='MpServer', x=-112.50, y=74.00, z=30.50]]
	Server brand: vanilla
	Server type: Non-integrated multiplayer server
Stacktrace:
	at crg.a(WorldClient.java:576)
	at cft.b(SourceFile:1815)
	at cft.a(SourceFile:403)
	at net.minecraft.client.main.Main.main(SourceFile:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.13.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 557587056 bytes (531 MB) / 1308622848 bytes (1248 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Launched Version: wwe 1.13.2
	LWJGL: 3.1.6 build 14
	OpenGL: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 411.63, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 6x AMD FX(tm)-6300 Six-Core Processor 
	OptiFine Version: OptiFine_1.13.2_HD_U_E6
	OptiFine Build: 20190214-140229
	Render Distance Chunks: 48
	Mipmaps: 4
	Anisotropic Filtering: 16
	Antialiasing: 0
	Multitexture: true
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 411.63
	OpenGlRenderer: GeForce GTX 1060 6GB/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 6`

**Comment by Vonr** [comment]
Not a WWE problem, there's a bat that is crashing your game at coords -485.11, 30.10, -11.40
